### PR TITLE
put workflows in proper fs directory

### PR DESCRIPTION
**Pull request by btylerburton**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+task :build do
+  system('bundle exec jekyll build')
+end
+
+desc "Check for broken links"
+task :brokenLinks do
+  puts "Checking for broken links..."
+  system('bundle exec jekyll build')
+  system('bundle exec htmlproofer --check-html _site')
+end
```
[the remark this answers is not in the document]
Comment (btylerburton) [reply Rakefile]: it's a ruby task runner.

[the remark this answers is not in the document]
Comment (nickumia-reisys) [reply Rakefile]: I guess what's the benefit we're trying to add by using it?

File: Rakefile [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+desc "Build the Jekyll site"
+task :build do
+  system('bundle exec jekyll build')
+end
+
+task :testLinks do
+  sh "bundle exec jekyll build"
+  options = { swap_urls: "^/BASEURL/:/" }
+  HTMLProofer.check_directory("./_site", options).run
```
Comment (nickumia-reisys) [inline Rakefile:19]: Is necessary because of the federalist bug of links being broken?  Or is this for something else?

**Comment by btylerburton** [comment]
ready to merge once this is merged: https://github.com/GSA/data.gov/pull/4123

File: README.md [hunk 3]
```diff
@@ -16,25 +16,15 @@ U.S. Data Federation website
...
-    $ bundle exec htmlproofer --check-html _site
+After you've confirmed the above requirements, all lifecycle operations can be run with Make commands (ex `make install`).
+
+| Make Command | Alias for                                    | Description                                                |
+| ------------ | -------------------------------------------- | ---------------------------------------------------------- |
+| install      | `bundle install`                             | Install dependencies (pinned in Gemfile.lock, if it exists |
+| update       | `bundle update`                              | Update installed dependencies; Reacreate Gemfile.lock      |
+| build        | `bundle exec jekyll build`                   | Build the site                                             |
+| serve        | `bundle exec jekyll serve`                   | Serve a local build                                        |
+| testlinks    | `bundle exec htmlproofer --check-html _site` | Check for broken links                                     |
```
Comment (nickumia-reisys) [inline README.md:27]: Is the alias important?  I feel like it's too much detail for the README

Comment (btylerburton) [reply README.md:27]: i'm fine with taking them out.

**Comment by btylerburton** [comment]
Looks like remote workflows need to specify a branch @nickumia-reisys 
- https://github.com/GSA/us-data-federation/actions/runs/3698810027/workflow 
- https://stackoverflow.com/questions/73026597/github-actions-suddenly-throwing-no-version-specified-for-reusable-workflows-t

**Review by nickumia-reisys — APPROVED** [review]
Makes sense ☁️

**Pull request merged**